src/lib.rs

```diff
@@ -389,14 +386,14 @@ fn send_detach_message(unix_stream_stdin: &mut UnixStream) -> anyhow::Result<()>
 #[logfn(Debug)]
 #[logfn_inputs(Debug)]
 fn run_client(unix_stream: &mut UnixStream, escape_code: u8) -> anyhow::Result<()> {
-    let mut _stdout = stdout().into_raw_mode()?;
```

Comment:
well, so `_stdout` was unused, however it looks like `into_raw_mode()` has a desired side-effect?

Reply:
yes, correct

Reply:
I've changed it to use just one variable and make the notion more explicit and it's working.